pcornet/v3.1/constraints/primary_keys.csv

```diff
@@ -16,7 +16,6 @@ pcornet,v3.1,primary key,pcornet_trial,patid,xpk_pcornet_trial
 pcornet,v3.1,primary key,pcornet_trial,trialid,xpk_pcornet_trial
 pcornet,v3.1,primary key,pcornet_trial,participantid,xpk_pcornet_trial
 pcornet,v3.1,primary key,death,patid,xpk_death
-pcornet,v3.1,primary key,death,death_date,xpk_death
```

Comment:
@bruth Could you confirm this change when you can?  According to page 70 of the [specs ](http://pcornet.org/wp-content/uploads/2016/11/2016-11-15-PCORnet-Common-Data-Model-v3.1_Specification.pdf),  it was removed as a constraint.